src/hooks/useENSProfileForm.ts

```diff
@@ -9,33 +10,48 @@ import {
 } from '@rainbow-me/redux/ensRegistration';
 import { AppState } from '@rainbow-me/redux/store';
 
+const selectedFieldsAtom = atom({
```

Comment:
had to take a look to `recoil` we barely use it on the app but we do use it on `src/App.js` from this PR and some other hooks https://github.com/rainbow-me/rainbow/pull/2627

wondering if we want to start using it more broadly @osdnk

Reply:
I don't uderstand why this is here in favor of jest state? However, if this is something that should persist during the session I guess this is easier for small states than redux. No strong opinions, we will review this at some point while making a bigger refactor of state management.